test/browser.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const assert = require('assert');
+const wd = require('wd');
+const util = require('./util');
+const abab = require('..').btoa;
+const atob = abab.atob;
+const btoa = abab.btoa;
+
+const browsers = ['chrome', 'firefox', 'internet explorer'];
+
+if (process.env.SAUCE_USER) {
```

Comment:
It looks like Sauce uses `SAUCE_USERNAME` instead of `SAUCE_USER`. I think it might be more straightforward if we made this match the Sauce lingo.